extended_tck.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+docker rm -f cloudstate-proxy
+docker rm -f cloudstate-function
+docker rm -f cloudstate-function-client
+#  fresh docker build
+docker build -t dev-cloudstate-tck:local ./
+
+#  primary tck tests for shopping cart
+docker run -d --name cloudstate-function -p 8080:8080 dev-cloudstate-tck:local \
+    server \
+    shoppingcart
+sleep 10
+docker run -d --name cloudstate-proxy -p 9000:9000 \
+    -e USER_FUNCTION_HOST=host.docker.internal \
+    -e USER_FUNCTION_PORT=8090 \
+    cloudstateio/cloudstate-proxy-dev-mode
+sleep 10
+docker run --rm --name cloudstate-tck -p 8090:8090 \
+    -e TCK_HOST=0.0.0.0 \
+    -e TCK_PROXY_HOST=host.docker.internal \
+    -e TCK_FRONTEND_HOST=host.docker.internal cloudstateio/cloudstate-tck
+
+status=$?
+echo "Removing docker containers"
+docker rm -f cloudstate-proxy
+docker rm -f cloudstate-function
+
+#  secondary integration tests for stateless function:
+docker run -d --name cloudstate-proxy -p 9000:9000 \
+    -e USER_FUNCTION_HOST=host.docker.internal \
+    -e USER_FUNCTION_PORT=8080 \
+    cloudstateio/cloudstate-proxy-dev-mode
+sleep 10
+docker run -d --name cloudstate-function -p 8080:8080 dev-cloudstate-tck:local \
+    server \
+    functiondemo \
+    shoppingcart
+sleep 10
+docker run --name cloudstate-function-client dev-cloudstate-tck:local \
+    client \
+    functiondemo \
+    shoppingcart
+
+docker rm -f cloudstate-proxy
```

Comment:
I liked the naming of containers to be independent from anything already existing and its cleanup no matter what happens during a run from the deleted `tck/run_tck.sh`

Reply:
I restored the unique container name functionality

> > Hi @chomnoue,
> > Thank you for your contribution! We really value the time you've taken to put this together.
> > Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:
> > https://www.lightbend.com/contribute/cla
> 
> @GratefulTony I see now the typesafe-cla-validator mentioning "chomnoue" for commits you probably pushed. Also there are quite few unrelated commits I think visible in this branch. Could you rebase your changes/additions somehow cleanly to your branch? Regarding the CLA Validator if I remember correctly you got approved its check with your initial commit for this PR, or is this still pending? Thanks :)



> > Hi @chomnoue,
> > Thank you for your contribution! We really value the time you've taken to put this together.
> > Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:
> > https://www.lightbend.com/contribute/cla
> 
> @GratefulTony I see now the typesafe-cla-validator mentioning "chomnoue" for commits you probably pushed. Also there are quite few unrelated commits I think visible in this branch. Could you rebase your changes/additions somehow cleanly to your branch? Regarding the CLA Validator if I remember correctly you got approved its check with your initial commit for this PR, or is this still pending? Thanks :)

Yeah. I got approved, but had to go back and rewrite some of my earlier commits since I had my email misconfigured in git config on a new dev machine. I'm in the middle of a rebase to try to clean things up. this is my best guess as to why chomnoue is getting flagged.

Reply:
I restored the uniquely-named containers